source/_components/media_player.denonavr.markdown

```diff
@@ -68,5 +72,6 @@ A few notes:
 - Additional option the control Denon AVR receivers with a builtin web server is using the HTTP interface with denonavr platform.
 - denonavr platform supports some additional functionalities like album covers, custom input source names and auto discovery.
 - Marantz receivers seem to a have quite simliar interface. Thus if you own one, give it a try.
+- The key-value structure in the sound_mode_dict is needed because the commands to set an sound mode and the reported sound mode are diffrent. This structure matches the reported sound mode with the commands to set a sound mode.
```

Comment:
- `diffrent` -> `different`
- Backquote sound_mode_dict keyword

Reply:
How do you make a backquote in this "code"?